src/ChannelServer/Network/ChannelPacketHandler.cs

```diff
@@ -806,49 +806,72 @@ public void CZ_CLIENT_HIT_LIST(ChannelConnection conn, Packet packet)
 		public void CZ_REQ_NORMAL_TX_NUMARG(ChannelConnection conn, Packet packet)
 		{
 			var size = packet.GetShort();
-			var type = packet.GetShort(); // 1 - statpints, 2 - skillpoints
-			var count = packet.GetInt();
-			//Throws exception when trying distributing skill points (count 6)
-			if (count != 5)
-				throw new Exception("Unknown CZ_REQ_NORMAL_TX_NUMARG format, expected 5 stats.");
```

Comment:
Not sure if you should throw new exceptions vs debug logging. Haven't looked trough aura source lately to confirm though.. @exectails ?